mypy/fscache.py

```diff
@@ -149,6 +149,11 @@ def md5(self, path: str) -> str:
             self.read(path)
         return self.hash_cache[path]
 
+    def samefile(self, f1: str, f2: str) -> bool:
+        s1 = self.stat(f1)
+        s2 = self.stat(f2)
+        return os.path.samestat(s1, s2)  # type: ignore
```

Comment:
I'm curious -- why do you need `# type: ignore` here?

Reply:
We were missing this function in the stub. We just merged the PR adding it, so this ignore can be removed as soon as we sync typeshed.